docs/troubleshooting/installation.md

```diff
@@ -24,15 +24,17 @@ Users can press the key combination `CTRL + ALT + F2` to switch to another TTY a
 
 ## Stuck in `Loading images. This may take a few minutes...`
 
-- Because the system doesn't have a default route, your installer may become "stuck" in this state. You can check your route status by executing the following command:
+Because the system doesn't have a default route, your installer may become "stuck" in this state. You can check your route status by executing the following command:
 
 ```shell
 $ ip route
 default via 10.10.0.10 dev mgmt-br proto dhcp        <-- Does a default route exist?
 10.10.0.0/24 dev mgmt-br proto kernel scope link src 10.10.0.15
 ```
 
-- Check that your DHCP server offers a default route option. Attaching content from `/run/cos/target/rke2.log` is helpful too.
+Check that your DHCP server offers a default route option. Attaching content from `/run/cos/target/rke2.log` is helpful too.
+
+Refer [dhcp server options](../install/pxe-boot-install.md#dhcp-server-configuration) for more details.
```

Comment:
```suggestion
For more information, see [DHCP Server Configuration](./pxe-boot-install.md#dhcp-server-configuration).
```

Reply:
Those two files are in different folders, use `../install/...` to refer is required.

[docs/troubleshooting/installation.md]

[docs/install/pxe-boot-install.md]